### PR TITLE
[BEAM-2829] Unit test for label pipeline option

### PR DESCRIPTION
**Pull request by aaltay**
R: @reuvenlax

### CODE REVIEW

**Comment by aaltay** [comment]
R: @charlesccychen

File: sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py [hunk 1]
```diff
@@ -173,6 +173,28 @@ def test_harness_override_absent_in_unreleased_sdk(self):
...
+         '--temp_location', 'gs://test-location/temp'])
+    job = apiclient.Job(pipeline_options)
+    self.assertIsNone(job.proto.labels)
+
+    pipeline_options = PipelineOptions(
+        ['--project', 'test_project', '--job_name', 'test_job_name',
+         '--temp_location', 'gs://test-location/temp',
+         '--label', 'key1=value1',
+         '--label', 'key2',
+         '--label', 'key3=value3'])
```
Comment (charlesccychen) [inline sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py]: Can we test the "--labels" alias too?

**Comment by aaltay** [comment]
Thank you @charlesccychen PTAL.

File: sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py (hunk 1 above)
Comment (charlesccychen) [inline sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py]: > **charlesccychen** wrote:
> Can we test the "--labels" alias too?


Done.

**Pull request closed without merging**